frontend/src/utils/helpers.js

```diff
@@ -49,6 +49,8 @@ export const formatDate = (dateString) =>
   dateString !== "None" &&
   moment(dateString, "YYYY-MM-DD").format("MMM DD YYYY");
 
+export const formatPostalCode = (code) => code && code.replace(/.{3}$/, " $&");
```

Comment:
Why is this formatting needed?

Reply:
The backend expects only 6 ch - `V6V4H4`
The UI would like to display a space between `V6V 4H4`